File: CS/LoadDocumentFromStream/Form1.cs
using System.IO;
using System.Windows.Forms;

namespace LoadDocumentFromStream {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            // Load a document from the stream.
            FileStream stream = new FileStream("..\\..\\Demo.pdf", FileMode.Open);
            pdfViewer1.LoadDocument(stream);
        }
    }
}

File: VB/LoadDocumentFromStream/Form1.vb
Imports System.IO
Imports System.Windows.Forms

Namespace LoadDocumentFromStream
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

            ' Load a document from the stream.
            Dim stream As New FileStream("..\..\Demo.pdf", FileMode.Open)
            pdfViewer1.LoadDocument(stream)
        End Sub
    End Class
End Namespace



